package.json

```diff
@@ -61,7 +61,7 @@
     "typedi": "^0.8.0",
     "typeorm": "^0.2.25",
     "typeorm-typedi-extensions": "^0.2.3",
-    "typescript": "^3.9.5",
+    "typescript": "^4.0.0",
```

Comment:
This is to pick up the `??=` operator (see [here](https://github.com/microsoft/TypeScript/issues/37255)).

Reply:
this is god tier ty for sharing

Reply:
@trulyronak check out the TypeScript roadmap ([here](https://github.com/Microsoft/TypeScript/wiki/Roadmap)) for all sorts of nifty things.